setup/bundles.rst

```diff
@@ -173,7 +173,7 @@ Before diving into the specifics of the most common edge cases, the general
 recommendation is to **not rely on the Symfony Kernel version** to decide which
 code to use::
 
-    if (Kernel::VERSION_ID <= 20800) {
+    if (Kernel::VERSION_ID < 20800) {
```

Comment:
@wouterj I don't think it is relevant, because this will be true for both 2.7 and 2.8, right? And the code change should generally apply when switching from 2.7 to 2.8, instead of between 2 and 3, otherwise you get deprecation notices for 2.8.